{{cookiecutter.project_slug}}/Makefile

```diff
@@ -103,25 +103,38 @@ tests-unit tests-integration:
 
 tests: tests-unit tests-integration ## runs unit and integration tests
 
-#----------------------------------- oSparc service versioning
+
+
+# PUBLISHING ----------------------------------- 
+define _bumpversion
+	# upgrades as $(subst $(1),,$@) version, commits and tags
+	@bump2version --verbose --list $(subst $(1),,$@)
+endef
+
 .PHONY: version-service-patch version-service-minor version-service-major
 version-service-patch version-service-minor version-service-major: versioning/service.cfg ## kernel/service versioning as patch
-	@bump2version --verbose  --list --config-file $< $(subst version-service-,,$@)
-#---------------------------------------- oSparc service publishing
+	@$(call _bumpversion,version-service-)
+
+.PHONY: version-integration-patch version-integration-minor version-integration-major
+version-integration-patch version-integration-minor version-integration-major: versioning/integration.cfg ## integration versioning as patch (bug fixes not affecting API/handling), minor/major (backwards-compatible/INcompatible API changes)
+	@$(call _bumpversion,version-integration-)
+
+

```

Comment:
I disagree with this move. The idea that it is further down in the Makefile is because one should not change the integration version that often. On the top of the Makefile (and therefore the help) are the things most often used.